packages/workbox-build/src/lib/utils/filter-files.js

```diff
@@ -89,7 +89,8 @@ module.exports = (fileDetails, options) => {
 
   if (options.manifestTransforms) {
     if (Array.isArray(options.manifestTransforms)) {
-      manifestTransforms.concat(options.manifestTransforms);
+      manifestTransforms =
+          manifestTransforms.concat(options.manifestTransforms);
```

Comment:
i added it, just in case you want it: https://github.com/GoogleChrome/workbox/pull/715 

Feel free to toss it if you'd rather wait.